bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java

```diff
@@ -52,14 +53,51 @@ public void testSendMany() throws IOException {
             targets.put(TLTC_TEST_FAUCET_ADDRESS, amount);
             amount = amount.add(initAmount);
         }
-//        targets.put("[ADDRESS2]", new BigDecimal("0.001"));
-        Optional<SendCoinsResponse> resp = client.sendMany(COIN, WALLET_ID, WALLET_PASSPHRASE, targets, null,
-                "test", null, null, 1, true);
+        parameters.put("coin", COIN);
+        parameters.put("walletId", WALLET_ID);
+        parameters.put("walletPass", WALLET_PASSPHRASE);
+        parameters.put("recipients", targets);
+        parameters.put("message", "test");
+        parameters.put("minConfirms", 1);
+        parameters.put("enforceMinConfirmsForChange", true);
+        parameters.put("sequenceId", ""); //Set it up for transaction to work
+        Optional<SendCoinsResponse> resp = client.sendMany(parameters);
         Assert.assertTrue(resp.isPresent());
         Assert.assertNotNull(resp.get().getTx());
 
     }
 
+    @Test
+    @Ignore
+    public void testSendManyFail() throws IOException {
+        Map<String, Object> parameters = new HashMap<String, Object>();
+        int unlockResult = client.unlock("0000000", TimeUnit.HOURS.toSeconds(1));
+        System.out.println(unlockResult);
+        assertTrue(unlockResult != 400 && unlockResult != 401);
+        Map<String, BigDecimal> targets = new HashMap<>();
+        BigDecimal initAmount = new BigDecimal("0.001").movePointRight(8);
+        BigDecimal amount = initAmount;
+        targets.put(TLTC_TEST_FAUCET_ADDRESS, amount);
+        // avoid coin required parameter to test error handling
+        parameters.put("coin", "");
+        parameters.put("walletId", WALLET_ID);
+        parameters.put("walletPass", WALLET_PASSPHRASE);
```

Comment:
There needs to be a test with a missing parameter (not an _empty_ one like here, but no key for that parameter at all).